benten-starter/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM tomcat:9-jre8-alpine
```

Comment:
@kritiagrawal112 can you add instructions on how to start the application using docker in the README ?

Reply:
@DivakarUngatla addressed the comment